src/pages/contact/ContactUs.module.scss

```diff
@@ -0,0 +1,96 @@
+@import 'global.scss';
+
+.Contact_Us {
```

Comment:
.FeaturedServices, .Text, .Break, .Images, .Caption and among others are not used here. Copying and pasting has its place, but if you do it, please remove code that's not used!